### ISSUE TITLE
[Bug] Batch Normalization Initializer not used

### ISSUE THREAD

**Issue by benjamin-lucas** [keras-team/keras#13214]
I have encountered a bug whereby the moving_mean initializer in the batch normalization layer is not being used.

**System information**  
- Have I written custom code (as opposed to using example directory):  Yes
- OS Platform and Distribution: Linux Ubuntu 16.04 
- TensorFlow backend (yes / no): Yes
- TensorFlow version: 1.13.1
- Keras version: 2.2.4
- Python version: 3.6.8
- CUDA/cuDNN version: 10.0.130

**Describe the current behavior** 
The (untrainable) moving_mean parameter does not use the initial value in calculations.

In the example below, the first model has the moving mean initialized with zeros and the second model has it initialized with ones. After training each model on one data point they both have the same value for the moving_mean parameter.

The same thing occurs with the moving_var. It appears that in each situation, the initializer is being ignored.

**Describe the expected behavior** 
I would expect that after training on one batch of data, the moving_mean parameter should be:

_updated_value_=_0.99*_initial_value_+_0.01_*_mean_of_training_batch

In the example below, we would expect to see significantly different values for the two models (closer to their respective initial values).

**Code to reproduce the issue**  
import numpy as np
import keras
from keras import layers
from keras import optimizers
from keras.models import Model
from keras.layers import Input, BatchNormalization, Dense

data = np.array([1])
label = np.array([[1,0]])

x = layers.Input(shape=(1,))
dense = layers.Dense(1, kernel_initializer="ones")(x)
batchnorm = layers.BatchNormalization(momentum=0.99, moving_mean_initializer="zeros")(dense)
out = layers.Dense(2, activation="softmax")(batchnorm)
model_zeros = Model(inputs = [x], outputs = [out])
model_zeros.compile(loss="categorical_crossentropy", optimizer=optimizers.Adam())

print("Model initialized with zeros")
print("Before training, the moving mean parameter of the BN layer is: ", model_zeros.layers[2].get_weights()[2][0])

model_zeros.fit(data, label, verbose=0)

print("After training, the moving mean parameter of the BN layer is: ", model_zeros.layers[2].get_weights()[2][0])

x = layers.Input(shape=(1,))
dense = layers.Dense(1, kernel_initializer="ones")(x)
batchnorm = layers.BatchNormalization(momentum=0.99, moving_mean_initializer="ones")(dense)
out = layers.Dense(2, activation="softmax")(batchnorm)
model_ones = Model(inputs = [x], outputs = [out])
model_ones.compile(loss="categorical_crossentropy", optimizer=optimizers.Adam())

print("Model initialized with ones")
print("Before training, the moving mean parameter of the BN layer is: ", model_ones.layers[2].get_weights()[2][0])

model_ones.fit(data, label, verbose=0)

print("After training, the moving mean parameter of the BN layer is: ", model_ones.layers[2].get_weights()[2][0])

As this produces the same result as the previous model, we can see that the initial value of the moving_mean is not being used.
